daemons/fenced/fenced_commands.c

```diff
@@ -2748,17 +2748,22 @@ bool fencing_peer_active(crm_node_t *peer)
 
 void set_fencing_completed(remote_fencing_op_t * op)
 {
-#ifdef CLOCK_MONOTONIC
-    struct timespec tv;
+    bool have = false;
```

Comment:
do we really need that?
something like
```
...
} else
#endif 
{
...
}
```
should do - right?

Reply:
now that i look at it again, i can simply return if we get it